docs/conf/conf.py

```
@@ -1,11 +1,15 @@
-from datetime import date
+# ruff: noqa: INP001 Configuration file for the Sphinx documentation builder.
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `INP001`)

Remove unused `noqa` directive

(RUF100)